pom.xml

```diff
@@ -297,7 +297,7 @@
         <!-- travis of July 2018 was still using _151 -->
         <build.jdk.minimum>1.8.0-151</build.jdk.minimum>
         <!-- travis of July 2018 was still using maven-3.5.2 -->
```

Comment:
the comment needs to be updated.

Reply:
I removed the comments as they seemed redundant with the actual variables and the version Travis uses should not enforce what we allow. Right ?

Reply:
I think we want the comments there. We need a way to know who we're waiting on to raise the version.

For example these comments would tell us that it is `3.3.8` because of the docker build. That makes it easy to go check that first when we want to raise it. (It saves a lot of time over having to run CI tests in branches, even in master,  to see if there's a possibility to raise the version.